src/common/components/profile-menu/index.tsx

```diff
@@ -55,6 +55,20 @@ export class ProfileMenu extends Component<Props> {
         })
       ]
     };
+    let secondaryMenu = [
+      {
+        label: _t(`profile.section-trail`),
+        href: `/@${username}/trail`,
+        selected: filter === "trail",
+        id: "trail"
+      }
+    ];
+    const kebabMenuConfig = {
```

Comment:
May be kebab menu is not even needed. IMHO adding a tab in same manner as other tabs would be a better approach. It has following benefits:

- Easy management of responsiveness in tabs and converting them to a dropdown on mobile screens. 
- Highlighting the currently active tabs. Currently Trail is highlighted inside the kebab menu and is not visible if trail is selected. The user does not know if he is viewing the trail page. 

The kebab menu is not responsive. One mobile screens there are two dropdowns, one is kebab menu and other one is the tabs on mobile screens. This does not look suitable. See screenshot for better understanding.

![image](https://github.com/ecency/ecency-vision/assets/106739598/a1fa0cd1-5190-4dce-b00e-6ae3b1934ea1)